docs/coding-guidelines/package-projects.md

```diff
@@ -94,7 +94,7 @@ Sample \ref .builds file defining a constant used to filter API that were added
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.7</NuGetTargetMoniker>
-    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp1.1'">$(DefineConstants);netcoreapp11</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp1.1'">$(DefineConstants);netcoreapp11</DefineConstants>
```

Comment:
This whole document should be updated. Also does the `TargetFramework`  comparison here actually work with the new config system which includes the OSGroup?

Reply:
Yeah, by the time TargetFramework hits the project, the OSGroup is removed.

Reply:
I dont have full context on different cases we handle in our pkgproj. so i am leaving this one to another PR. I will probably talk with someone before updating the docs